WrathCombo/Combos/PvP/RPRPVP.cs

```diff
@@ -127,36 +113,37 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                         // Outside of Enshroud
                         if (!enshrouded)
                         {
-
                             if (HasEffect(Buffs.PerfectioParata))
                                 return OriginalHook(TenebraeLemurum);
 
-                            // Death Warrant Option
-                            if (IsEnabled(CustomComboPreset.RPRPvP_Burst_DeathWarrant) &&
-                                deathWarrantReady && distance <= 25 &&
-                                ((plentifulCD > 20 && immortalStacks < immortalThreshold) ||
-                                (plentifulReady && immortalStacks >= immortalThreshold)) || HasEffect(Buffs.DeathWarrant) && GetBuffRemainingTime(Buffs.DeathWarrant) <= 3)
-                                return OriginalHook(DeathWarrant);
+                            // Pooling Plentiful with Death warrant
+                            if (IsEnabled(CustomComboPreset.RPRPvP_Burst_ImmortalPooling))
+                            {
+                                if (IsEnabled(CustomComboPreset.RPRPvP_Burst_DeathWarrant) && deathWarrantReady && distance <= 25 &&
+                                (GetCooldownRemainingTime(PlentifulHarvest) > 20 ||     //if plentiful will be back for the next death warrant
+                                (plentifulReady && immortalStacks >= immortalThreshold) || // if plentiful is ready for this death warrant and you have the charges you want
```

Comment:
Since this is a `>=` instead of an `>`, it seems to me like this would push users under their set thresholds.

Reply:
The set that they want to use it at 5, then 5 needs to be an option. if there was no =, it would only use it at 6 stacks.

Reply:
I believe it is referring to this pictured option, which is the # to save, not the # at which to use stacks.

I read this as "set this to a number we should always stay over", which would be a `>` sign.

![image](https://github.com/user-attachments/assets/5e847e82-f7c2-441a-b696-c2b5394c2bda)

Reply:
I think a wording tweak would be in order to keep the `=`.\
Or a lower maximum for the range, to switch to `>`.

Neither really affects anything, it's just my opinion, but the latter is closer to breaking user settings so maybe just some sort of tweaked wording; it's still an approval for me with or without this change - feel free to mark this as resolved.